app/controllers/orders_controller.rb

```diff
@@ -1,6 +1,6 @@
 class OrdersController < ApplicationController
   before_action :set_order, only: [:show, :edit, :update, :destroy]
-  # before_action :require_user
+  before_action :require_user
```

Comment:
Almost for each controller, you have added this

*before_action :require_user*

Why dont you move this to the application controller

Reply:
ok